pyproject.toml

```diff
@@ -79,6 +80,7 @@ profiling = [
 ]
 s3 = [
     "aiobotocore>=1.1.2",
+    "aioitertools>=0.7.0",
```

Comment:
Same with aioitertools here. They are not a direct dependency of the package itself.